sklearn/neighbors/_base.py

```diff
@@ -581,11 +588,11 @@ def kneighbors(self, X=None, n_neighbors=None, return_distance=True):
             In this case, the query point is not considered its own neighbor.
 
         n_neighbors : int
-            Number of neighbors to get (default is the value
-            passed to the constructor).
+            Number of neighbors required for each sample.
+            (default is the value passed to the constructor).
```

Comment:
there was two version of this line `(default is the value passed to the constructor).` one with `.. is the value ..` the other with `.. is value ..`